gui/storage/admin.py

```diff
@@ -344,6 +344,12 @@ def get_actions(self):
         )
 
         # Dataset actions
```

Comment:
The codepath to promote Dataset or Zvol seems to be the same.

Is there a way to make a single url/action to promote it?

Reply:
@william-gr that was the original way I did it, but I thought having one url would be fine because it doesn't have much of a difference, since both are datasets

Reply:
Thats why I am asking, why two different actions? One for Zvol and one for Dataset?